cs10/august/Convertor.java

```diff
@@ -0,0 +1,29 @@
+public class Convertor {
+
+    public boolean[] dec2bin(int decimal) {
+        boolean[] answer = new boolean[8];
+        int idx = 0;
+
+        while (decimal > 0) {
+            answer[idx++] = (decimal % 2) == 1 ? true : false;
```

Comment:
```suggestion
            answer[idx++] = (decimal % 2) == 1;
```

`(decimal % 2) == 1`
이 부분은 `boolean isOdd = (decimal % 2) == 1`
이런식으로 별도의 변수로 빼는게 가독성이 더 좋을 것 같습니다.